metrics/mdl_high_water_mark

```diff
@@ -1 +1 @@
-5
+8
```

Comment:
This doesn't look right with both a banner image and a top-level markdown header, but markdownlint both really doesn't want raw HTML and also doesn't want a missing top-level markdown header.  Oh well.